dotCMS/src/curl-test/Content_Report_Resource.postman_collection.json

```diff
@@ -221,7 +222,71 @@
 								"site",
 								"default"
 							]
+						},
+						"description": "Generates a report using the Site Key."
+					},
+					"response": []
+				},
+				{
+					"name": "Generate Report - Default Site With ID",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"HTTP Status code must be 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"pm.test(\"Checking that there at least 2 Content Types living under the Default Site\", function () {",
+									"    const entity = pm.response.json().entity;",
+									"    pm.expect(entity.length).to.be.gte(2, \"There must be at least 2 Content Types living under the Default Site\");",
+									"});",
+									"",
+									"pm.test(\"Checking that pagination information matches the specified parameters\", function () {",
+									"    const pagination = pm.response.json().pagination;",
+									"    pm.expect(pagination.perPage).equals(10, \"The 'perPage' parameter must be 10\");",
+									"    pm.expect(pagination.totalEntries).to.be.gte(2, \"The 'totalEntries' parameter must be at least 2\");",
+									"});",
+									""
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
 						}
+					],
+					"request": {
+						"auth": {
```

Comment:
Let's use bearer token where instead of basic from now on. 
Take a look here https://github.com/dotCMS/core/pull/27780/files

Reply:
Done! I'm using the JWT for the Postman test now.